packages/block-editor/src/hooks/block-bindings-panel.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { store as blocksStore } from '@wordpress/blocks';
+import {
+	PanelBody,
+	__experimentalItemGroup as ItemGroup,
+	__experimentalItem as Item,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../store';
+import { unlock } from '../lock-unlock';
+import InspectorControls from '../components/inspector-controls';
+
+export const BlockBindingsPanel = () => {
+	const { block } = useSelect( ( select ) => {
+		const { getSelectedBlockClientId, getBlock } = unlock(
```

Comment:
I don't think these selectors are locked. This needs to be verified.

Reply:
Thanks—I removed the unnecessary lock